packaging/nnstreamer.spec

```diff
@@ -345,11 +350,12 @@ export NNSTREAMER_DECODERS=$(pwd)/build/ext/nnstreamer/tensor_decoder
 %define test_script $(pwd)/packaging/run_unittests_binaries.sh
 
 %if %{with tizen}
+    %ifarch %arm aarch64
```

Comment:
This is not needed. The test_script handles if the file exists or not.

Reply:
updated